core/cursor_svg.js

```diff
@@ -237,6 +243,37 @@ Blockly.CursorSvg.prototype.showWithField_ = function() {
   this.showCurrent_();
 };
 
+/**
+ * Show the cursor using a stack.
+ * @private
+ */
+Blockly.CursorSvg.prototype.showWithStack_ = function() {
+  var block = this.getCurNode().getLocation();
+
+  //Gets the height and width of entire stack
```

Comment:
General formatting comment: single line comments should have a space after the second slash, and a period at the end.  As usual for this sort of stuff, apply it to the lines you touched here and change existing ones as you touch them in other PRs.